pkg/chartmuseum/server/multitenant/handlers.go

```diff
@@ -186,6 +183,34 @@ func (server *MultiTenantServer) getStorageObjectTemplateRequestHandler(c *gin.C
 	})
 }
 
+func (server *MultiTenantServer) getStorageObjectRawRequestHandler(c *gin.Context) {
+	repo := c.Param("repo")
+	name := c.Param("name")
+	version := c.Param("version")
+
+	log := server.Logger.ContextLoggingFn(c)
+
+	fileName, err := server.getChartFileName(log, repo, name, version)
+	if err != nil {
+		c.JSON(http.StatusNotFound, gin.H{"error": err.Message})
+		return
+	}
+
+	storageObject, err := server.getStorageObject(log, repo, fileName)
+	if err != nil {
+		c.JSON(err.Status, gin.H{"error": err.Message})
+		return
+	}
+	chrt, err1 := loader.LoadArchive(bytes.NewReader(storageObject.Content))
```

Comment:
Can we just put this in `pkg/repo/chart.go`, and provide a function like `GetChartRawdate()` ?

Reply:
thank you for your review
I tried to wrap a function like you said, but I got an error `import cycle not allowed`
That would require a bigger change, what's your opinion? whether this larger change is needed